src/types.ts

```diff
@@ -300,6 +300,16 @@ export interface OptionsObject<V extends VariantType = VariantType> extends Shar
      * @default false
      */
     persist?: boolean;
+    /**
+     * Order of a display. The order is switched only in the snackbar that is already displayed.
+     * @default 0
+     */
+     displayOrder?: number;
+    /**
+     * Reverse the order of a display. The reversed order is switched only in the snackbar that is already displayed.
+     * @default false
+     */
+    reverse?: boolean;
```

Comment:
Wouldn't this be unnecessary by adjusting the `displayOrder`?

e.g.
Assuming display in order of size
displayOrder  set to 
1. Always on top (call): 1000000000000000 or higher (1000000000000000 ms is over 30000 years)
2. reverse order (notification):  1000000000000000 - Date.now() 
3. Show under notification(notificationViewAll): 1
4. others: displayOrder  0 

This will not be a problem until 30,000 years from now.

Reply:
// This will not be a problem until 30,000 years from now.
Sounds good to me.

Reply:
@satoren I was testing this last night, and I am not really sure how to set the reverse order for the notification snacks. The order of display works just fine, but the reversing is the one that I am not totally able to grasp on how to do it? 

In your comment here https://github.com/oviceinc/notistack/pull/1#discussion_r1299502647, do you mean to alternate between the below 2 based on the `displayOrder` value?
```typescript
            const [nomineeSnack] = snacks.sort((a, b) => a.displayOrder - b.displayOrder);
            const [nomineeSnack] = snacks.sort((a, b) => b.displayOrder - a.displayOrder);
```

Reply:
It means that the displayOrder is set in reverse order on the side using the library.
If you want to order by date, add the appropriate standard value to `Date.now()`.
If you want to reverse the order of the date, just subtract  `Date.now()` from the time far enough away.

Reply:
Okay, I will test again. Thank you for the example.

Reply:
@satoren 

Seems working correctly using the below code. 

```typescript
// SnackProvider.tsx
const [nomineeSnack] = snacks.sort((a, b) => b.displayOrder - a.displayOrder);

// SnackbarComponent.tsx (React Component)
const notificationDisplayOrderMin = 1000;
const notificationDisplayOrderRangeSize =
  new Date(30000, 0).getTime() - new Date(0, 0).getTime();
const notificationDisplayOrderMax =
  notificationDisplayOrderMin + notificationDisplayOrderRangeSize;

function computeNotificationDisplayOrder(date: Date) {
  return (
    (date.getTime() % notificationDisplayOrderRangeSize) +
    notificationDisplayOrderMin
  );
}

const callDisplayOrderNew = notificationDisplayOrderMax + 1;
const notificationDisplayOrder = computeNotificationDisplayOrder(new Date());
const viewMoreDisplayOrder = notificationDisplayOrderMin - 1; 
const regularDisplayOrderNew = viewMoreDisplayOrder - 1;
```
https://github.com/oviceinc/notistack/assets/58634448/4a0a913a-17f5-46a0-890b-41fa9784838a

Reply:
@satoren let me know what you think, then I will proceed by reverting the latest commit in here [Pass reverse property to InternalSnack](https://github.com/oviceinc/notistack/pull/1/commits/b5a71aca4e87693361cf6e5467bd3e632d552f67) and update this https://github.com/oviceinc/notistack/pull/1#discussion_r1299502647

Also, one question when can we migrate to https://github.com/oviceinc/ovice-ui/pull/2788 v3?

Reply:
Yes, the reverse property has the problem that it is hard to imagine how it works, so it is better not to add it.
> Also, one question when can we migrate to https://github.com/oviceinc/ovice-ui/pull/2788 v3?

Can you please follow [this](https://notistack.com/migration) guide to migrate to v3?
Use this repository after the migration.

Reply:
Understood, thank you.